### DISCUSSION TITLE
Welcome to gotchiverse-bible Discussions!

### DISCUSSION THREAD

**Discussion by cinnabarhorse** [aavegotchi/gotchiverse-bible#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by elfif** [comment]
Hello @cinnabarhorse, not much happened here, but i would like to ask a question about attack traits.
Specifically about melee damage vs range damage traits value and  hypothetical conversion into quantified damages.
In most RPG / ARPG games, it ends up being quantified in DPS (damage per second). If I apply the same logic here, calculations _could_ be (that's my 50 cents ofc) : 

- Melee DPS: Melee damage * Attack speed
- Ranged DPS: Ranged damage * Attack speed 

Thing is Melee DPS is based on AGG and BRN traits and is non constant. Looks like: 
`AGG * (100 - BRN)`

Let's say i have AGG = 90 and BRN = 10, DPS = 8100. I hit hard and fast
Let's say i have AGG = 10 and BRN = 90, DPS = 100. I hit low and slow

On the other hand Ranged DPS is purely based on BRN trait and looks pretty constant:
`BRN * (100 - BRN)`

Let's say i have BRN = 90, DPS = 900 I hit hard but slow.
Let's say i have BRN = 10, DPS = 900 I hit low and fast.

Even if my calculation is inaccurate, i just want to point the fact that it looks like melee build may have some adavantages over ranged in terms of build potentials, based on my understanding of current system.
